src/axom/spin/examples/spin_introduction.cpp

```diff
@@ -430,54 +426,40 @@ void showImplicitGrid()
 
 void makeTriangles(std::vector<TriangleType>& tris)
 {
-  PointType p[8];
-  p[0] = PointType::make_point(0.3, 0.93, 0.03);
-  p[1] = PointType::make_point(0.1, 0.85, 0.01);
-  p[2] = PointType::make_point(0.3, 0.78, 0.03);
-  p[3] = PointType::make_point(0.18, 0.36, 0.018);
-  p[4] = PointType::make_point(0.8, 0.58, 0.08);
-  p[5] = PointType::make_point(0.6, 0.5, 0.06);
-  p[6] = PointType::make_point(0.55, 0.42, 0.055);
-  p[7] = PointType::make_point(0.61, 0.1, 0.061);
-
-  TriangleType t0(p[0], p[1], p[2]);
-  tris.push_back(t0);
-  TriangleType t1(p[2], p[1], p[3]);
-  tris.push_back(t1);
-  TriangleType t2(p[2], p[3], p[6]);
-  tris.push_back(t2);
-  TriangleType t3(p[6], p[3], p[7]);
-  tris.push_back(t3);
-  TriangleType t4(p[4], p[2], p[6]);
-  tris.push_back(t4);
-  TriangleType t5(p[4], p[5], p[7]);
-  tris.push_back(t5);
+  PointType p[8] = {PointType {0.3, 0.93, 0.03},    // 0
+                    PointType {0.1, 0.85, 0.01},    // 1
+                    PointType {0.3, 0.78, 0.03},    // 2
+                    PointType {0.18, 0.36, 0.018},  // 3
+                    PointType {0.8, 0.58, 0.08},    // 4
+                    PointType {0.6, 0.5, 0.06},     // 5
+                    PointType {0.55, 0.42, 0.055},  // 6
+                    PointType {0.61, 0.1, 0.061}};  // 7
+
+  tris.emplace_back(TriangleType(p[0], p[1], p[2]));
```

Comment:
I think you can just do `tris.emplace_back(p[0], p[1], p[2]);` to make it even simpler.

Reply:
Thanks for the suggestion -- I updated these in the files I touched